src/analyzer.rs

```diff
@@ -1402,4 +1405,33 @@ type: object
         let structs = analyze(schema, "Reference", Cfg::default()).unwrap().0;
         assert_eq!(structs[0].members[0].type_, "Option<Vec<ObjectReference>>");
     }
+
+    #[test]
+    fn array_of_array() {
+        init();
+        let schema_str = r#"
+      properties:
+        mounts:
+          description: mounts specifies a list of mount points to be setup.
+          items:
+            description: MountPoints defines input for generated mounts in cloud-init.
+            items:
+              type: string
+            type: array
+          type: array
+      type: object
+      "#;
+
+        let schema: JSONSchemaProps = serde_yaml::from_str(schema_str).unwrap();
+
+        let structs = analyze(schema, "KubeadmConfig", Cfg::default()).unwrap().0;
+
+        let root = &structs[0];
+        assert_eq!(root.name, "KubeadmConfig");
+        assert_eq!(root.level, 0);
+
+        let map = &root.members[0];
+        assert_eq!(map.name, "mounts");
+        assert_eq!(map.type_, "Option<Vec<Vec<String>>>");
```

Comment:
test makes sense based on the schema and as a sanity, it does seem to match what i [find in the wild for the crd](https://github.com/AdminTurnedDevOps/kubernetes-examples/blob/9d6e5c214b0a1ad18cacba8888605a239a4775aa/azure-capi/capi-azurekubeadm.yaml#L101-L103)